md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -104,7 +104,7 @@ By default the [WildFly bundle](wildfly-bundle.md) is configured to accept conne
 
 For an EJB3 client connection to work with WildFly 9, there are some additional configuration steps.
 
-If the client is not located on the same host than the WildFly server, you need to create a Application user in the WildFly administration console.
+If the client is not located on the same host than the WildFly server, you need to create a Application user in the WildFly administration console.  
```

Comment:
Zut! I remove MAAAANY of them, without knowing they had functional meaning :-(
My bad. We should restore them, thus.